kafka-streams/src/main/java/io/micronaut/configuration/kafka/streams/health/KafkaStreamHealthSerdes.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.configuration.kafka.streams.health;
+
+import io.micronaut.serde.annotation.SerdeImport;
+import org.apache.kafka.streams.processor.TaskId;
+
+
+/**
+ * Additional Serdes for Kafka streams health types returned as part of the {@link io.micronaut.management.health.indicator.HealthResult}.
+ */
+@SerdeImport(TaskId.class)
+public class KafkaStreamHealthSerdes {
```

Comment:
thanks can you make this non-public and add `@Internal`?

Reply:
thanks, a test would be good if possible

Reply:
I had written one, but wasn't sure if it was too specific a test to use a `JsonMapper` to check if a `TaskId` is serializable. If you think its appropriate here I'll add it. 

The kafka-streams instances injected into the tests have 0 active or standby tasks which is why this was never caught. 

Lmk your thoughts on this. If you want the concrete test I'll add it

Reply:
just a check that the type is serialisable is fine

Reply:
While testing I've realised that the `TaskId` isn't serializable out of the box with Jackson, and if it were it would yield:
```
"taskId": {
  "topicGroupId": 0,
  "partition": 0
}
```
but I'm not sure this is helpful as part of the health report however. 

The `TaskId` object has a custom `.toString()` which formats it in a readable manner. I think its more appropriate here.
```
"taskId": "0_0"
```

After all, the task id is meant to be an identifier, showing its component internal fields in this manner doesn't seem helpful. 

This would be the change, adding the `toString()`. I'd make it null safe of course
```
details.put("taskId", taskMetadata.taskId().toString());
```

Thoughts?

Reply:
sure, for that case you probably need to implement a `Serializer` and remove the `@SerdeImport`

Reply:
Well I was just going to call `.toString()` on the task object that gets put into the `Map<String, Object>`. 

Or would you rather a custom Serializer to allow users to override the serializer if they want?